docs/source/development/style-guide/requirement-and-issue-tracking.rst

```diff
@@ -0,0 +1,120 @@
+Name                Definition
+=================== ===================================================================================================================================================================================================================================================
+Level 4 requirement requirements that have been decomposed to the SDC level. Level 4 requirements have already been defined.
+Level 5 requirement requirements decomposed from the level 4 requirements that better define how the SDC plans to satisfy the parent level 4 requirement, while not defining any implementation details. Example: Backup *Metadata* to AWS backup account vs. Backup *OpenSearch* to AWS backup account
+Implementation task Github issue that better defines how a certain feature will be implemented. Implementation tasks are tracked under the parent level 5 requirement as a sub-task. Each implementation task should be small enough to be handled in a single PR.
+SPIKE               Github issue created to investigate different architecture design approaches. Like implementation tasks, SPIKEs should be tracked under the applicable level 5 requirement as a sub-task.
+=================== ===================================================================================================================================================================================================================================================
+
+Requirements Overview
+^^^^^^^^^^^^^^^^^^^^^
+
+Requriements Mapping
+~~~~~~~~~~~~~~~~~~~~
+
+* Each level 4 requirement is mapped to at least one SIT.
+* Each level 4 requirement relevant to the upcoming SIT is decomposed into level 5 requirements that, together, cover the testing requirements of the SIT.
+* The level 5 requirement creation is incremental as we move through development for each SIT.
+* Each Level 5 requirement will have implementation tasks (sub-issues) that fully cover the Level 5 requirement and contain implementation specific information.
+
+.. image:: ../../_static/req-decomposition.png
+   :alt: Requirement decomposition diagram
+   :width: 1293
+   :height: 629
+
+Requirements Lifecycle
+~~~~~~~~~~~~~~~~~~~~~~
+The below image shows an example of the lifecycle of a Level 4 requirement over the course of the project:
+
+.. image:: ../../_static/req-lifecycle.png
+   :alt: Requirement lifecycle diagram
+   :width: 1631
+   :height: 621
+
+
+Overview of Managing Requirements in Github
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* All level 5 requirements will be managed as Github issues.
+* The status of the requirement issue ("Open" / "Closed") is an indication of both development and testing status, so a requirement is not "Closed" until it is implemented and has passed a SIT.
+* Requirement issues should track sub-issues that describe the implementation.
```

Comment:
How should they be tracked? With a checklist of issues is how we've done it before, but would be good to call it out. Those checklists also provide a visual indicator of how "finished" a task is, so we can see if the requirement is complete even without relying on open/closed.

Reply:
I see this is already covered in the "creating level 5 requirements" section, so I am ok with leaving this as-is